frontend/catalyst/oqc/oqc_device.py

```diff
@@ -36,7 +36,7 @@ class OQCDevice(Device):
     """The OQC device allows to access the hardware devices from OQC using
     Catalyst."""
 
-    config = pathlib.Path(__file__).parent.joinpath("oqc.toml")
+    config = pathlib.Path(__file__).parent.joinpath("/src/oqc.toml")
```

Comment:
```suggestion
    config = pathlib.Path(__file__).parent / "src" / "oqc.toml"
```

Reply:
@rmoyard for some reason, joining path with something starting with "/" will lead you to start again. I think using `.joinpath("src/oqc.toml")` may have worked as well.

Reply:
Feel free to commit thanks!